CHANGELOG.md

```diff
@@ -6,6 +6,20 @@ Deprecated:
 - Authorization using Role-Based Access Control
 - LDAP Embedded Support
 
+## v0.93.0
+
+:new: What's new:
+
+- Export gob registration to let external services to use auth.Claims (#5184)
```

Comment:
`gob`?

Reply:
Yes, we use [encoding/gob](https://pkg.go.dev/encoding/gob), not sure why.  It's called a "gob":

> Package gob manages streams of gobs

Added some words.